### ISSUE TITLE
 DigitalOcean key generation

### ISSUE THREAD

**Issue by axos88** [ansible/ansible-modules-core#607]
##### Issue Type:

Bug Report
##### Component Name:

digital_ocean_sshkey
##### Ansible Version:

ansible 1.8.2
  configured module search path = None
##### Environment:

Mac OS X
##### Summary:

If an SSH key exists in digitalocean, trying to "state=present" the same SSH public key with another name fails
##### Steps To Reproduce:
- Create an SSH key on the WebUI, and name it key1
- run this ansible task:

```
local_action: digital_ocean_sshkey state=present name=key2 ssh_pub_key={{digitalocean_ssh_key_file}} client_id={{digitalocean_client_id}} api_key={{digitalocean_api_key}}
```
##### Expected Results:

One of:
- An explanatory error message
- the key being renamed
- the id of the already existing key being returned (possibly with the name key1)
##### Actual Results:

```
failed: [127.0.0.1 -> 127.0.0.1] => {"failed": true}
msg: SSH Key failed to be created
```

**Comment by ansibot** [comment]
@mgregson, @alukovenko, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by alukovenko** [comment]
needs_info
@axos88, what version of dopy do you use?

**Comment by axos88** [comment]
not sure what version i used in january 2015... I moved on from DO in the meantime

**Comment by ansibot** [comment]
@mgregson, @alukovenko, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by alukovenko** [comment]
Seems to be caused by dopy, wontfix
